{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#Olympia Oyster transcriptome"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "See Geoduck page for details on how to download and check integretity of raw files.."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/sr320/data-genomic/tentacle/OlyOv5\n"
     ]
    }
   ],
   "source": [
    "cd /Users/sr320/data-genomic/tentacle/OlyOv5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/sr320/data-genomic/tentacle/OlyOv5/PE\n"
     ]
    }
   ],
   "source": [
    "cd PE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!cat \\\n",
    "filtered_106A_Female_Mix_GATCAG_L004_R1.fastq \\\n",
    "filtered_108A_Female_Mix_GGCTAC_L004_R1.fastq \\\n",
    "filtered_106A_Male_Mix_TAGCTT_L004_R1.fastq \\\n",
    "filtered_108A_Male_Mix_AGTCAA_L004_R1.fastq >> /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\n",
    "!cat \\\n",
    "filtered_106A_Female_Mix_GATCAG_L004_R2.fastq \\\n",
    "filtered_108A_Female_Mix_GGCTAC_L004_R2.fastq \\\n",
    "filtered_106A_Male_Mix_TAGCTT_L004_R2.fastq \\\n",
    "filtered_108A_Male_Mix_AGTCAA_L004_R2.fastq >> /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@HWI-ST700693:263:C0K6DACXX:4:1101:1464:1969\r\n",
      "NTTCATGAAGGGGTCAACCAGAATGATCTCGTAAAACTTGTAAGATGAATCCTGTGCCACCCAGTAAGAGCTTAGC\r\n",
      "+\r\n",
      "#4=DDDDDHFFHHGGABHIICBGH<E@FHIIGEGIIIIIIIDGGIIGEIIIIIIIIIIGIGGI9=D@CECEEF@CC\r\n",
      "@HWI-ST700693:263:C0K6DACXX:4:1101:1506:1985\r\n",
      "NCCTGATTGAAAACAAAATGTCTTTGTTTTTTCTACTTTGTATGGATCAATCAATCTTATTTGTTCCTGTAGAGAT\r\n",
      "+\r\n",
      "#1=DDFFFHHHHHJJJJJIGFHHJIIIIJJJJIIJIJJJJIJJFEGJIIIIHIGIJJEHIJJJGAGHJJIIIHGD;\r\n",
      "@HWI-ST700693:263:C0K6DACXX:4:1101:1882:1979\r\n",
      "NGGGACTCTAACGAGAACCTTCCGTCATCTGGCTCTGGACATAACCATGGTTAACAAGAACACTGTCATGGTGGAG\r\n"
     ]
    }
   ],
   "source": [
    "!head /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/sr320/data-genomic/tentacle/OlyOv6\n"
     ]
    }
   ],
   "source": [
    "cd /Users/sr320/data-genomic/tentacle/OlyOv6"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of OlyOv6_combined_R1.fastq\n",
      "Approx 5% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 10% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 15% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 20% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 25% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 30% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 35% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 40% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 45% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 50% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 55% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 60% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 65% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 70% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 75% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 80% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 85% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 90% complete for OlyOv6_combined_R1.fastq\n",
      "Approx 95% complete for OlyOv6_combined_R1.fastq\n",
      "Analysis complete for OlyOv6_combined_R1.fastq\n",
      "Started analysis of OlyOv6_combined_R2.fastq\n",
      "Approx 5% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 10% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 15% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 20% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 25% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 30% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 35% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 40% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 45% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 50% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 55% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 60% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 65% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 70% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 75% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 80% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 85% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 90% complete for OlyOv6_combined_R2.fastq\n",
      "Approx 95% complete for OlyOv6_combined_R2.fastq\n",
      "Analysis complete for OlyOv6_combined_R2.fastq\n"
     ]
    }
   ],
   "source": [
    "!/Applications/bioinfo/FastQC/fastqc \\\n",
    "/Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq \\\n",
    "/Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "These files are created locally\n",
    "\n",
    "<img src=\"http://eagle.fish.washington.edu/cnidarian/skitch/OlyOv6_1BBEE01A.png\" alt=\"OlyOv6_1BBEE01A.png\"/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<img src=\"http://eagle.fish.washington.edu/cnidarian/skitch/OlyOv6_combined_R1_fastq_FastQC_Report_1BBEE058.png\" alt=\"OlyOv6_combined_R1_fastq_FastQC_Report_1BBEE058.png\"/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<img src=\"http://eagle.fish.washington.edu/cnidarian/skitch/OlyOv6_combined_R1_fastq_FastQC_Report_1BBEE088.png\" alt=\"OlyOv6_combined_R1_fastq_FastQC_Report_1BBEE088.png\"/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<img src=\"http://eagle.fish.washington.edu/cnidarian/skitch/OlyOv6_combined_R1_fastq_FastQC_Report_1BBEE0AB.png\" alt=\"OlyOv6_combined_R1_fastq_FastQC_Report_1BBEE0AB.png\"/>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Similar results for R2\n",
    "\n",
    "<img src=\"http://eagle.fish.washington.edu/cnidarian/skitch/OlyOv6_combined_R2_fastq_FastQC_Report_1BBEE46A.png\" alt=\"OlyOv6_combined_R2_fastq_FastQC_Report_1BBEE46A.png\"/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "u'/Users/sr320/data-genomic/tentacle/OlyOv6'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Path to Cutadapt set as: '/Users/sr320/.local/bin/cutadapt' (user defined)\n",
      "1.8.1\n",
      "Cutadapt seems to be working fine (tested command '/Users/sr320/.local/bin/cutadapt --version')\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t5712\tAGATCGGAAGAGC\t1000000\t0.57\n",
      "Nextera\t3\tCTGTCTCTTATA\t1000000\t0.00\n",
      "smallRNA\t1\tATGGAATTCTCG\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 5712). Second best hit was Nextera (count: 3)\n",
      "\n",
      "Writing report to 'OlyOv6_combined_R1.fastq_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.0\n",
      "Cutadapt version: 1.8.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Length cut-off for read 1: 35 bp (default)\n",
      "Length cut-off for read 2: 35 bb (default)\n",
      "All Read 1 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "\n",
      "Writing final adapter and quality trimmed output to OlyOv6_combined_R1_trimmed.fq\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "This is cutadapt 1.8.1 with Python 2.7.10\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 2144.94 s (11 us/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             200,998,380\n",
      "Reads with adapters:                73,774,681 (36.7%)\n",
      "Reads written (passing filters):   200,998,380 (100.0%)\n",
      "\n",
      "Total basepairs processed: 15,275,876,880 bp\n",
      "Quality-trimmed:             416,684,162 bp (2.7%)\n",
      "Total written (filtered):  14,564,248,250 bp (95.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 73774681 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.0%\n",
      "  C: 25.2%\n",
      "  G: 17.2%\n",
      "  T: 20.3%\n",
      "  none/other: 4.3%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t52089917\t50249595.0\t0\t52089917\n",
      "2\t13122915\t12562398.8\t0\t13122915\n",
      "3\t3984745\t3140599.7\t0\t3984745\n",
      "4\t876041\t785149.9\t0\t876041\n",
      "5\t192459\t196287.5\t0\t192459\n",
      "6\t42209\t49071.9\t0\t42209\n",
      "7\t21249\t12268.0\t0\t21249\n",
      "8\t16482\t3067.0\t0\t16482\n",
      "9\t17084\t766.7\t0\t12797 4287\n",
      "10\t17718\t191.7\t1\t10499 7219\n",
      "11\t12818\t47.9\t1\t9549 3269\n",
      "12\t12839\t12.0\t1\t11800 1039\n",
      "13\t8626\t3.0\t1\t8184 442\n",
      "14\t8526\t3.0\t1\t8067 459\n",
      "15\t7023\t3.0\t1\t6596 427\n",
      "16\t7179\t3.0\t1\t6596 583\n",
      "17\t7496\t3.0\t1\t6726 770\n",
      "18\t7079\t3.0\t1\t6428 651\n",
      "19\t4954\t3.0\t1\t4399 555\n",
      "20\t5021\t3.0\t1\t4372 649\n",
      "21\t4882\t3.0\t1\t4090 792\n",
      "22\t5126\t3.0\t1\t4458 668\n",
      "23\t6242\t3.0\t1\t4749 1493\n",
      "24\t6742\t3.0\t1\t4742 2000\n",
      "25\t7701\t3.0\t1\t5052 2649\n",
      "26\t7470\t3.0\t1\t4266 3204\n",
      "27\t11441\t3.0\t1\t4746 6695\n",
      "28\t10960\t3.0\t1\t5263 5697\n",
      "29\t12888\t3.0\t1\t5044 7844\n",
      "30\t10274\t3.0\t1\t6240 4034\n",
      "31\t8909\t3.0\t1\t3674 5235\n",
      "32\t9062\t3.0\t1\t4115 4947\n",
      "33\t19175\t3.0\t1\t3926 15249\n",
      "34\t20247\t3.0\t1\t6814 13433\n",
      "35\t20215\t3.0\t1\t6308 13907\n",
      "36\t18977\t3.0\t1\t6897 12080\n",
      "37\t24248\t3.0\t1\t5779 18469\n",
      "38\t29759\t3.0\t1\t7077 22682\n",
      "39\t38372\t3.0\t1\t7309 31063\n",
      "40\t49968\t3.0\t1\t10540 39428\n",
      "41\t34797\t3.0\t1\t13808 20989\n",
      "42\t38698\t3.0\t1\t9281 29417\n",
      "43\t27563\t3.0\t1\t13132 14431\n",
      "44\t38729\t3.0\t1\t6563 32166\n",
      "45\t42853\t3.0\t1\t12030 30823\n",
      "46\t30309\t3.0\t1\t10213 20096\n",
      "47\t27693\t3.0\t1\t7937 19756\n",
      "48\t30221\t3.0\t1\t7209 23012\n",
      "49\t29430\t3.0\t1\t7957 21473\n",
      "50\t28966\t3.0\t1\t7287 21679\n",
      "51\t69734\t3.0\t1\t7286 62448\n",
      "52\t92152\t3.0\t1\t17407 74745\n",
      "53\t41276\t3.0\t1\t19721 21555\n",
      "54\t36134\t3.0\t1\t7428 28706\n",
      "55\t31643\t3.0\t1\t9177 22466\n",
      "56\t48430\t3.0\t1\t7663 40767\n",
      "57\t48431\t3.0\t1\t10921 37510\n",
      "58\t78974\t3.0\t1\t10519 68455\n",
      "59\t52248\t3.0\t1\t17284 34964\n",
      "60\t55257\t3.0\t1\t9701 45556\n",
      "61\t76855\t3.0\t1\t12860 63995\n",
      "62\t175186\t3.0\t1\t17609 157577\n",
      "63\t197887\t3.0\t1\t39008 158879\n",
      "64\t122848\t3.0\t1\t38522 84326\n",
      "65\t85413\t3.0\t1\t22646 62767\n",
      "66\t116361\t3.0\t1\t19925 96436\n",
      "67\t163900\t3.0\t1\t25021 138879\n",
      "68\t271961\t3.0\t1\t37785 234176\n",
      "69\t365011\t3.0\t1\t58318 306693\n",
      "70\t241610\t3.0\t1\t76299 165311\n",
      "71\t145508\t3.0\t1\t38669 106839\n",
      "72\t74901\t3.0\t1\t27598 47303\n",
      "73\t19037\t3.0\t1\t7591 11446\n",
      "74\t13889\t3.0\t1\t4672 9217\n",
      "75\t5707\t3.0\t1\t1628 4079\n",
      "76\t30031\t3.0\t1\t10128 19903\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\n",
      "=============================================\n",
      "200998380 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to 'OlyOv6_combined_R2.fastq_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.0\n",
      "Cutadapt version: 1.8.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Length cut-off for read 1: 35 bp (default)\n",
      "Length cut-off for read 2: 35 bb (default)\n",
      "All Read 1 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases\n",
      "All Read 2 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\n",
      "\n",
      "Writing final adapter and quality trimmed output to OlyOv6_combined_R2_trimmed.fq\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "This is cutadapt 1.8.1 with Python 2.7.10\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 2050.25 s (10 us/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             200,998,380\n",
      "Reads with adapters:                72,221,243 (35.9%)\n",
      "Reads written (passing filters):   200,998,380 (100.0%)\n",
      "\n",
      "Total basepairs processed: 15,275,876,880 bp\n",
      "Quality-trimmed:             582,620,358 bp (3.8%)\n",
      "Total written (filtered):  14,547,993,528 bp (95.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 72221243 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.2%\n",
      "  C: 26.2%\n",
      "  G: 17.6%\n",
      "  T: 21.1%\n",
      "  none/other: 0.9%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t53392369\t50249595.0\t0\t53392369\n",
      "2\t12692989\t12562398.8\t0\t12692989\n",
      "3\t3996044\t3140599.7\t0\t3996044\n",
      "4\t898465\t785149.9\t0\t898465\n",
      "5\t199957\t196287.5\t0\t199957\n",
      "6\t41733\t49071.9\t0\t41733\n",
      "7\t21420\t12268.0\t0\t21420\n",
      "8\t16937\t3067.0\t0\t16937\n",
      "9\t21594\t766.7\t0\t15418 6176\n",
      "10\t21338\t191.7\t1\t12772 8566\n",
      "11\t16835\t47.9\t1\t11422 5413\n",
      "12\t12411\t12.0\t1\t11193 1218\n",
      "13\t12316\t3.0\t1\t9998 2318\n",
      "14\t13123\t3.0\t1\t12314 809\n",
      "15\t7466\t3.0\t1\t6846 620\n",
      "16\t8648\t3.0\t1\t7523 1125\n",
      "17\t9515\t3.0\t1\t9033 482\n",
      "18\t5625\t3.0\t1\t5083 542\n",
      "19\t6811\t3.0\t1\t6422 389\n",
      "20\t5214\t3.0\t1\t4965 249\n",
      "21\t4196\t3.0\t1\t3988 208\n",
      "22\t5056\t3.0\t1\t4787 269\n",
      "23\t8323\t3.0\t1\t5755 2568\n",
      "24\t7914\t3.0\t1\t7268 646\n",
      "25\t6468\t3.0\t1\t5960 508\n",
      "26\t10420\t3.0\t1\t5889 4531\n",
      "27\t7100\t3.0\t1\t6000 1100\n",
      "28\t7244\t3.0\t1\t6611 633\n",
      "29\t5672\t3.0\t1\t4854 818\n",
      "30\t12359\t3.0\t1\t11824 535\n",
      "31\t2079\t3.0\t1\t1623 456\n",
      "32\t3282\t3.0\t1\t3072 210\n",
      "33\t2203\t3.0\t1\t2014 189\n",
      "34\t2939\t3.0\t1\t2686 253\n",
      "35\t3399\t3.0\t1\t3149 250\n",
      "36\t3761\t3.0\t1\t3478 283\n",
      "37\t3774\t3.0\t1\t3480 294\n",
      "38\t3903\t3.0\t1\t3569 334\n",
      "39\t4639\t3.0\t1\t4278 361\n",
      "40\t5311\t3.0\t1\t4954 357\n",
      "41\t6074\t3.0\t1\t5422 652\n",
      "42\t11033\t3.0\t1\t10481 552\n",
      "43\t3527\t3.0\t1\t3210 317\n",
      "44\t5255\t3.0\t1\t4529 726\n",
      "45\t7410\t3.0\t1\t6983 427\n",
      "46\t2498\t3.0\t1\t2316 182\n",
      "47\t3537\t3.0\t1\t3265 272\n",
      "48\t2722\t3.0\t1\t2518 204\n",
      "49\t3561\t3.0\t1\t3277 284\n",
      "50\t7011\t3.0\t1\t6634 377\n",
      "51\t5356\t3.0\t1\t5029 327\n",
      "52\t2795\t3.0\t1\t2571 224\n",
      "53\t2345\t3.0\t1\t2117 228\n",
      "54\t4249\t3.0\t1\t4023 226\n",
      "55\t5601\t3.0\t1\t5243 358\n",
      "56\t4440\t3.0\t1\t3896 544\n",
      "57\t5862\t3.0\t1\t5432 430\n",
      "58\t12815\t3.0\t1\t12283 532\n",
      "59\t9503\t3.0\t1\t9079 424\n",
      "60\t11281\t3.0\t1\t10794 487\n",
      "61\t16326\t3.0\t1\t15494 832\n",
      "62\t30462\t3.0\t1\t28919 1543\n",
      "63\t80723\t3.0\t1\t77509 3214\n",
      "64\t147721\t3.0\t1\t137737 9984\n",
      "65\t225595\t3.0\t1\t207184 18411\n",
      "66\t68475\t3.0\t1\t62667 5808\n",
      "67\t10044\t3.0\t1\t8962 1082\n",
      "68\t3200\t3.0\t1\t2861 339\n",
      "69\t1568\t3.0\t1\t1362 206\n",
      "70\t912\t3.0\t1\t808 104\n",
      "71\t836\t3.0\t1\t580 256\n",
      "72\t571\t3.0\t1\t458 113\n",
      "73\t787\t3.0\t1\t415 372\n",
      "74\t613\t3.0\t1\t495 118\n",
      "75\t1057\t3.0\t1\t938 119\n",
      "76\t4626\t3.0\t1\t4181 445\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq\n",
      "=============================================\n",
      "200998380 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files OlyOv6_combined_R1_trimmed.fq and OlyOv6_combined_R2_trimmed.fq\n",
      "file_1: OlyOv6_combined_R1_trimmed.fq, file_2: OlyOv6_combined_R2_trimmed.fq\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: OlyOv6_combined_R1_trimmed.fq and OlyOv6_combined_R2_trimmed.fq <<<<<\n",
      "Writing validated paired-end read 1 reads to OlyOv6_combined_R1_val_1.fq\n",
      "Writing validated paired-end read 2 reads to OlyOv6_combined_R2_val_2.fq\n",
      "\n",
      "Writing unpaired read 1 reads to OlyOv6_combined_R1_unpaired_1.fq\n",
      "Writing unpaired read 2 reads to OlyOv6_combined_R2_unpaired_2.fq\n",
      "\n",
      "Total number of sequences analysed: 200998380\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 7305383 (3.63%)\n",
      "\n",
      "Deleting both intermediate output files OlyOv6_combined_R1_trimmed.fq and OlyOv6_combined_R2_trimmed.fq\n",
      "\n",
      "====================================================================================================\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!/Applications/bioinfo/trim_galore_zip/trim_galore \\\n",
    "--paired \\\n",
    "--clip_R1 14 \\\n",
    "--clip_R2 14 \\\n",
    "--retain_unpaired \\\n",
    "--path_to_cutadapt /Users/sr320/.local/bin/cutadapt \\\n",
    "/Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq \\\n",
    "/Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "files generated\n",
    "<img src=\"http://eagle.fish.washington.edu/cnidarian/skitch/OlyOv6_1BBF222F.png\" alt=\"OlyOv6_1BBF222F.png\"/>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "SUMMARISING RUN PARAMETERS\r\n",
      "==========================\r\n",
      "Input filename: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\r\n",
      "Trimming mode: paired-end\r\n",
      "Trim Galore version: 0.4.0\r\n",
      "Cutadapt version: 1.8.1\r\n",
      "Quality Phred score cutoff: 20\r\n",
      "Quality encoding type selected: ASCII+33\r\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\r\n",
      "Maximum trimming error rate: 0.1 (default)\r\n",
      "Minimum required adapter overlap (stringency): 1 bp\r\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\r\n",
      "Length cut-off for read 1: 35 bp (default)\r\n",
      "Length cut-off for read 2: 35 bp (default)\r\n",
      "All Read 1 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases\r\n",
      "All Read 2 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\r\n",
      "\r\n",
      "\r\n",
      "This is cutadapt 1.8.1 with Python 2.7.10\r\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\r\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\r\n",
      "Finished in 2144.94 s (11 us/read; 5.62 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:             200,998,380\r\n",
      "Reads with adapters:                73,774,681 (36.7%)\r\n",
      "Reads written (passing filters):   200,998,380 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed: 15,275,876,880 bp\r\n",
      "Quality-trimmed:             416,684,162 bp (2.7%)\r\n",
      "Total written (filtered):  14,564,248,250 bp (95.3%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 73774681 times.\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-9 bp: 0; 10-13 bp: 1\r\n",
      "\r\n",
      "Bases preceding removed adapters:\r\n",
      "  A: 33.0%\r\n",
      "  C: 25.2%\r\n",
      "  G: 17.2%\r\n",
      "  T: 20.3%\r\n",
      "  none/other: 4.3%\r\n",
      "\r\n",
      "Overview of removed sequences\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "1\t52089917\t50249595.0\t0\t52089917\r\n",
      "2\t13122915\t12562398.8\t0\t13122915\r\n",
      "3\t3984745\t3140599.7\t0\t3984745\r\n",
      "4\t876041\t785149.9\t0\t876041\r\n",
      "5\t192459\t196287.5\t0\t192459\r\n",
      "6\t42209\t49071.9\t0\t42209\r\n",
      "7\t21249\t12268.0\t0\t21249\r\n",
      "8\t16482\t3067.0\t0\t16482\r\n",
      "9\t17084\t766.7\t0\t12797 4287\r\n",
      "10\t17718\t191.7\t1\t10499 7219\r\n",
      "11\t12818\t47.9\t1\t9549 3269\r\n",
      "12\t12839\t12.0\t1\t11800 1039\r\n",
      "13\t8626\t3.0\t1\t8184 442\r\n",
      "14\t8526\t3.0\t1\t8067 459\r\n",
      "15\t7023\t3.0\t1\t6596 427\r\n",
      "16\t7179\t3.0\t1\t6596 583\r\n",
      "17\t7496\t3.0\t1\t6726 770\r\n",
      "18\t7079\t3.0\t1\t6428 651\r\n",
      "19\t4954\t3.0\t1\t4399 555\r\n",
      "20\t5021\t3.0\t1\t4372 649\r\n",
      "21\t4882\t3.0\t1\t4090 792\r\n",
      "22\t5126\t3.0\t1\t4458 668\r\n",
      "23\t6242\t3.0\t1\t4749 1493\r\n",
      "24\t6742\t3.0\t1\t4742 2000\r\n",
      "25\t7701\t3.0\t1\t5052 2649\r\n",
      "26\t7470\t3.0\t1\t4266 3204\r\n",
      "27\t11441\t3.0\t1\t4746 6695\r\n",
      "28\t10960\t3.0\t1\t5263 5697\r\n",
      "29\t12888\t3.0\t1\t5044 7844\r\n",
      "30\t10274\t3.0\t1\t6240 4034\r\n",
      "31\t8909\t3.0\t1\t3674 5235\r\n",
      "32\t9062\t3.0\t1\t4115 4947\r\n",
      "33\t19175\t3.0\t1\t3926 15249\r\n",
      "34\t20247\t3.0\t1\t6814 13433\r\n",
      "35\t20215\t3.0\t1\t6308 13907\r\n",
      "36\t18977\t3.0\t1\t6897 12080\r\n",
      "37\t24248\t3.0\t1\t5779 18469\r\n",
      "38\t29759\t3.0\t1\t7077 22682\r\n",
      "39\t38372\t3.0\t1\t7309 31063\r\n",
      "40\t49968\t3.0\t1\t10540 39428\r\n",
      "41\t34797\t3.0\t1\t13808 20989\r\n",
      "42\t38698\t3.0\t1\t9281 29417\r\n",
      "43\t27563\t3.0\t1\t13132 14431\r\n",
      "44\t38729\t3.0\t1\t6563 32166\r\n",
      "45\t42853\t3.0\t1\t12030 30823\r\n",
      "46\t30309\t3.0\t1\t10213 20096\r\n",
      "47\t27693\t3.0\t1\t7937 19756\r\n",
      "48\t30221\t3.0\t1\t7209 23012\r\n",
      "49\t29430\t3.0\t1\t7957 21473\r\n",
      "50\t28966\t3.0\t1\t7287 21679\r\n",
      "51\t69734\t3.0\t1\t7286 62448\r\n",
      "52\t92152\t3.0\t1\t17407 74745\r\n",
      "53\t41276\t3.0\t1\t19721 21555\r\n",
      "54\t36134\t3.0\t1\t7428 28706\r\n",
      "55\t31643\t3.0\t1\t9177 22466\r\n",
      "56\t48430\t3.0\t1\t7663 40767\r\n",
      "57\t48431\t3.0\t1\t10921 37510\r\n",
      "58\t78974\t3.0\t1\t10519 68455\r\n",
      "59\t52248\t3.0\t1\t17284 34964\r\n",
      "60\t55257\t3.0\t1\t9701 45556\r\n",
      "61\t76855\t3.0\t1\t12860 63995\r\n",
      "62\t175186\t3.0\t1\t17609 157577\r\n",
      "63\t197887\t3.0\t1\t39008 158879\r\n",
      "64\t122848\t3.0\t1\t38522 84326\r\n",
      "65\t85413\t3.0\t1\t22646 62767\r\n",
      "66\t116361\t3.0\t1\t19925 96436\r\n",
      "67\t163900\t3.0\t1\t25021 138879\r\n",
      "68\t271961\t3.0\t1\t37785 234176\r\n",
      "69\t365011\t3.0\t1\t58318 306693\r\n",
      "70\t241610\t3.0\t1\t76299 165311\r\n",
      "71\t145508\t3.0\t1\t38669 106839\r\n",
      "72\t74901\t3.0\t1\t27598 47303\r\n",
      "73\t19037\t3.0\t1\t7591 11446\r\n",
      "74\t13889\t3.0\t1\t4672 9217\r\n",
      "75\t5707\t3.0\t1\t1628 4079\r\n",
      "76\t30031\t3.0\t1\t10128 19903\r\n",
      "\r\n",
      "\r\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R1.fastq\r\n",
      "=============================================\r\n",
      "200998380 sequences processed in total\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!cat OlyOv6_combined_R1.fastq_trimming_report.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "SUMMARISING RUN PARAMETERS\r\n",
      "==========================\r\n",
      "Input filename: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq\r\n",
      "Trimming mode: paired-end\r\n",
      "Trim Galore version: 0.4.0\r\n",
      "Cutadapt version: 1.8.1\r\n",
      "Quality Phred score cutoff: 20\r\n",
      "Quality encoding type selected: ASCII+33\r\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\r\n",
      "Maximum trimming error rate: 0.1 (default)\r\n",
      "Minimum required adapter overlap (stringency): 1 bp\r\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\r\n",
      "Length cut-off for read 1: 35 bp (default)\r\n",
      "Length cut-off for read 2: 35 bp (default)\r\n",
      "All Read 1 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases\r\n",
      "All Read 2 sequences will be trimmed by 14 bp from their 5' end to avoid poor qualities or biases (e.g. M-bias for BS-Seq applications)\r\n",
      "\r\n",
      "\r\n",
      "This is cutadapt 1.8.1 with Python 2.7.10\r\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq\r\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\r\n",
      "Finished in 2050.25 s (10 us/read; 5.88 M reads/minute).\r\n",
      "\r\n",
      "=== Summary ===\r\n",
      "\r\n",
      "Total reads processed:             200,998,380\r\n",
      "Reads with adapters:                72,221,243 (35.9%)\r\n",
      "Reads written (passing filters):   200,998,380 (100.0%)\r\n",
      "\r\n",
      "Total basepairs processed: 15,275,876,880 bp\r\n",
      "Quality-trimmed:             582,620,358 bp (3.8%)\r\n",
      "Total written (filtered):  14,547,993,528 bp (95.2%)\r\n",
      "\r\n",
      "=== Adapter 1 ===\r\n",
      "\r\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 72221243 times.\r\n",
      "\r\n",
      "No. of allowed errors:\r\n",
      "0-9 bp: 0; 10-13 bp: 1\r\n",
      "\r\n",
      "Bases preceding removed adapters:\r\n",
      "  A: 34.2%\r\n",
      "  C: 26.2%\r\n",
      "  G: 17.6%\r\n",
      "  T: 21.1%\r\n",
      "  none/other: 0.9%\r\n",
      "\r\n",
      "Overview of removed sequences\r\n",
      "length\tcount\texpect\tmax.err\terror counts\r\n",
      "1\t53392369\t50249595.0\t0\t53392369\r\n",
      "2\t12692989\t12562398.8\t0\t12692989\r\n",
      "3\t3996044\t3140599.7\t0\t3996044\r\n",
      "4\t898465\t785149.9\t0\t898465\r\n",
      "5\t199957\t196287.5\t0\t199957\r\n",
      "6\t41733\t49071.9\t0\t41733\r\n",
      "7\t21420\t12268.0\t0\t21420\r\n",
      "8\t16937\t3067.0\t0\t16937\r\n",
      "9\t21594\t766.7\t0\t15418 6176\r\n",
      "10\t21338\t191.7\t1\t12772 8566\r\n",
      "11\t16835\t47.9\t1\t11422 5413\r\n",
      "12\t12411\t12.0\t1\t11193 1218\r\n",
      "13\t12316\t3.0\t1\t9998 2318\r\n",
      "14\t13123\t3.0\t1\t12314 809\r\n",
      "15\t7466\t3.0\t1\t6846 620\r\n",
      "16\t8648\t3.0\t1\t7523 1125\r\n",
      "17\t9515\t3.0\t1\t9033 482\r\n",
      "18\t5625\t3.0\t1\t5083 542\r\n",
      "19\t6811\t3.0\t1\t6422 389\r\n",
      "20\t5214\t3.0\t1\t4965 249\r\n",
      "21\t4196\t3.0\t1\t3988 208\r\n",
      "22\t5056\t3.0\t1\t4787 269\r\n",
      "23\t8323\t3.0\t1\t5755 2568\r\n",
      "24\t7914\t3.0\t1\t7268 646\r\n",
      "25\t6468\t3.0\t1\t5960 508\r\n",
      "26\t10420\t3.0\t1\t5889 4531\r\n",
      "27\t7100\t3.0\t1\t6000 1100\r\n",
      "28\t7244\t3.0\t1\t6611 633\r\n",
      "29\t5672\t3.0\t1\t4854 818\r\n",
      "30\t12359\t3.0\t1\t11824 535\r\n",
      "31\t2079\t3.0\t1\t1623 456\r\n",
      "32\t3282\t3.0\t1\t3072 210\r\n",
      "33\t2203\t3.0\t1\t2014 189\r\n",
      "34\t2939\t3.0\t1\t2686 253\r\n",
      "35\t3399\t3.0\t1\t3149 250\r\n",
      "36\t3761\t3.0\t1\t3478 283\r\n",
      "37\t3774\t3.0\t1\t3480 294\r\n",
      "38\t3903\t3.0\t1\t3569 334\r\n",
      "39\t4639\t3.0\t1\t4278 361\r\n",
      "40\t5311\t3.0\t1\t4954 357\r\n",
      "41\t6074\t3.0\t1\t5422 652\r\n",
      "42\t11033\t3.0\t1\t10481 552\r\n",
      "43\t3527\t3.0\t1\t3210 317\r\n",
      "44\t5255\t3.0\t1\t4529 726\r\n",
      "45\t7410\t3.0\t1\t6983 427\r\n",
      "46\t2498\t3.0\t1\t2316 182\r\n",
      "47\t3537\t3.0\t1\t3265 272\r\n",
      "48\t2722\t3.0\t1\t2518 204\r\n",
      "49\t3561\t3.0\t1\t3277 284\r\n",
      "50\t7011\t3.0\t1\t6634 377\r\n",
      "51\t5356\t3.0\t1\t5029 327\r\n",
      "52\t2795\t3.0\t1\t2571 224\r\n",
      "53\t2345\t3.0\t1\t2117 228\r\n",
      "54\t4249\t3.0\t1\t4023 226\r\n",
      "55\t5601\t3.0\t1\t5243 358\r\n",
      "56\t4440\t3.0\t1\t3896 544\r\n",
      "57\t5862\t3.0\t1\t5432 430\r\n",
      "58\t12815\t3.0\t1\t12283 532\r\n",
      "59\t9503\t3.0\t1\t9079 424\r\n",
      "60\t11281\t3.0\t1\t10794 487\r\n",
      "61\t16326\t3.0\t1\t15494 832\r\n",
      "62\t30462\t3.0\t1\t28919 1543\r\n",
      "63\t80723\t3.0\t1\t77509 3214\r\n",
      "64\t147721\t3.0\t1\t137737 9984\r\n",
      "65\t225595\t3.0\t1\t207184 18411\r\n",
      "66\t68475\t3.0\t1\t62667 5808\r\n",
      "67\t10044\t3.0\t1\t8962 1082\r\n",
      "68\t3200\t3.0\t1\t2861 339\r\n",
      "69\t1568\t3.0\t1\t1362 206\r\n",
      "70\t912\t3.0\t1\t808 104\r\n",
      "71\t836\t3.0\t1\t580 256\r\n",
      "72\t571\t3.0\t1\t458 113\r\n",
      "73\t787\t3.0\t1\t415 372\r\n",
      "74\t613\t3.0\t1\t495 118\r\n",
      "75\t1057\t3.0\t1\t938 119\r\n",
      "76\t4626\t3.0\t1\t4181 445\r\n",
      "\r\n",
      "\r\n",
      "RUN STATISTICS FOR INPUT FILE: /Users/sr320/data-genomic/tentacle/OlyOv6/OlyOv6_combined_R2.fastq\r\n",
      "=============================================\r\n",
      "200998380 sequences processed in total\r\n",
      "\r\n",
      "Total number of sequences analysed for the sequence pair length validation: 200998380\r\n",
      "\r\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 7305383 (3.63%)\r\n"
     ]
    }
   ],
   "source": [
    "!cat OlyOv6_combined_R2.fastq_trimming_report.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Started analysis of OlyOv6_combined_R1_val_1.fq\n",
      "Approx 5% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 10% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 15% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 20% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 25% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 30% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 35% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 40% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 45% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 50% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 55% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 60% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 65% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 70% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 75% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 80% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 85% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 90% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Approx 95% complete for OlyOv6_combined_R1_val_1.fq\n",
      "Analysis complete for OlyOv6_combined_R1_val_1.fq\n",
      "Started analysis of OlyOv6_combined_R2_val_2.fq\n",
      "Approx 5% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 10% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 15% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 20% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 25% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 30% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 35% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 40% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 45% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 50% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 55% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 60% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 65% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 70% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 75% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 80% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 85% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 90% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Approx 95% complete for OlyOv6_combined_R2_val_2.fq\n",
      "Analysis complete for OlyOv6_combined_R2_val_2.fq\n"
     ]
    }
   ],
   "source": [
    "!/Applications/bioinfo/FastQC/fastqc \\\n",
    "OlyOv6_combined_R1_val_1.fq \\\n",
    "OlyOv6_combined_R2_val_2.fq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "#Assembly\n",
    "```\n",
    "Trinity.pl \\\n",
    "--seqType fq \\\n",
    "-JM 24G \\\n",
    "--left /Volumes/web/cnidarian/OlyOv6_combined_R1_val_1.fq \\\n",
    "--right /Volumes/web/cnidarian/OlyOv6_combined_R2_val_2.fq \\\n",
    "--CPU 16\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
